### PR TITLE
PowerShell integration for Front Door service

### PR DESCRIPTION
**Pull request by hytao**
## Description

1. Old PR against Private Repo https://github.com/Azure/azure-powershell-pr/pull/395
2. PowerShell integration for Azure FrontDoor service.
3. Design Review link: Azure/azure-powershell-cmdlet-review-pr#132 Azure/azure-powershell-cmdlet-review-pr#165

## Checklist

- [x ] I have read the [_Submitting Changes_](https://github.com/Azure/azure-powershell/blob/preview/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](https://github.com/Azure/azure-powershell/blob/preview/CONTRIBUTING.md)
- [ ] The title of the PR is clear and informative
- [ ] The appropriate [change log has been updated](https://github.com/Azure/azure-powershell/blob/preview/CONTRIBUTING.md#updating-the-change-log)
- [ ] The PR does not introduce [breaking changes](https://github.com/Azure/azure-powershell/blob/preview/documentation/breaking-changes/breaking-changes-definition.md)
- [ ] If applicable, the changes made in the PR have proper test coverage
- [ ] For public API changes to cmdlets:
    - [ ] the changes have gone through a [cmdlet design review](https://github.com/Azure/azure-powershell-cmdlet-review-pr)
    - [ ] the cmdlet markdown files were [generated using the `platyPS` module](https://github.com/Azure/azure-powershell/blob/preview/documentation/development-docs/help-generation.md)

### CODE REVIEW

**Comment by maddieclayton** [comment]
@hytao I assume your SDK has now been published, so can you remove the two FrontDoor nuget packages from the LocalFeed?

**Comment by adxsdkps** [comment]
Can one of the admins verify this patch?

File: src/ResourceManager/Profile/Commands.Profile/AzureRmAlias/Mappings.cs [hunk 1]
```diff
@@ -1,4 +1,4 @@
-// ----------------------------------------------------------------------------------
+﻿// ----------------------------------------------------------------------------------
```
Comment (maddieclayton) [inline src/ResourceManager/Profile/Commands.Profile/AzureRmAlias/Mappings.cs]: Please remove all changes to this file.  This is done automatically now.

**Comment by maddieclayton** [comment]
@azuresdkci Add to whitelist

**Comment by maddieclayton** [comment]
@hytao Looks good to me! Merging now.

**Pull request merged**